eng/pipelines/coreclr/templates/helix-queues-setup.yml

```
@@ -48,7 +48,7 @@ jobs:
 
     # Android x64
     - ${{ if in(parameters.platform, 'android_x64') }}:
-      - Ubuntu.1804.Amd64.Android.29.Open
+      - Ubuntu.2204.Amd64.Android.29.Open
```

The Android helix queues are only available on Ubuntu 18.04. There are no Ubuntu 22.04-based Android queues.

```suggestion
      - Ubuntu.1804.Amd64.Android.29.Open
```

---

I don't disbelieve you, but if that's the case, why is this PR green?

---

I think we need to run the runtime-extra-platforms CI legs to test these.

---

@radical @lewing are there other CI legs we should run for mono/wasm that could be affected by this, and are not executed by default?

---

Yeah we don't run the Android legs in the runtime pipeline. They're only run in runtime-extra-platforms or runtime-extra-platforms-android due to cost and reliability problems.

---

Understand now.